modules/content/sanity-content.service.ts

```diff
@@ -77,12 +83,12 @@ export class SanityContentService implements ContentService {
 
             await prisma.prismaToken.upsert({
                 where: {
-                    address_chain: { address: tokenAddress, chain: networkContext.chain },
+                    address_chain: { address: tokenAddress, chain: this.chain },
```

Comment:
isn't the chain returned in the response? is so we could use it here instead of setting it up externally